### PR TITLE
Content Updates

### PR DESCRIPTION
**Pull request by aphelionz**
This PR implements a number of content updates for the single rustipfs.com page. Each commit describes what it is, but the featured attraction here is the updates to the *Getting Started* section.

My goal with the Getting Started is that I'm trying to emulate the README `ipfs cat` command that is shown after you install and init go-ipfs. I'm open to suggestions about other things we can put for Getting Started, so long as they're roughly this number of lines (or fewer) in length.

As for the code itself, I ran it locally and I _believe it works_, but just needs a bootstrap node to connect to so it can discover the CID in the code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -90,47 +90,37 @@ <h3>Canonical HTTP APIs</h3>
...
-// ipfs = { git = "https://github.com/rs-ipfs/rust-ipfs.git" }
-// tokio = { version = "0.2", features = ["full"] }
-
 #[tokio::main]
 async fn main() {
-    let options = IpfsOptions::default();
+    // Load in some default testing options
+    let options = IpfsOptions::inmemory_with_generated_keys();
+
+    // TODO: Bootstrap
```
Comment (koivunej) [inline index.html]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -90,47 +90,37 @@ <h3>Canonical HTTP APIs</h3>
...
 
     // Start daemon and initialize repo
-    let (ipfs, fut) = UninitializedIpfs::<TestTypes>::new(options, None)
-        .await
-        .start()
-        .await
-        .unwrap();
+    let (ipfs, fut): (Ipfs&lt;TestTypes&gt;, _) = UninitializedIpfs::new(options).start().await.unwrap();
 
     tokio::spawn(fut);
```
Comment (koivunej) [inline index.html]:
```suggestion

ipfs.restore_bootstrappers().await.unwrap();

```

**Comment by aphelionz** [comment]
Updated the getting started code example, got it down to 45 lines so I'm happy with that. I can update the Cargo.toml code when ready. No rush

**Review by koivunej — COMMENTED** [review]
And we should soon be able to use this (this suggestion prompted me to find https://github.com/rs-ipfs/rust-ipfs/issues/405)

**Comment by aphelionz** [comment]
Looks like index.html was outdated when you left the content changes, so I couldn't commit. Check out the latest commit?

File: index.html [hunk 3]
```diff
@@ -90,55 +90,49 @@ <h3>Canonical HTTP APIs</h3>
...
-use ipfs::{IpfsOptions, IpfsPath, TestTypes, UninitializedIpfs};
-use std::convert::TryFrom;
+      <p>
+        First, add <code>ipfs</code> and <code>tokio</code>
+        to your <strong>Cargo.toml</strong> file:
+      </p>
+      <pre data-enlighter-language="toml">
+[dependencies]
+futures = { default-features = false, version = "0.3.5", features = ["alloc", "std"] }
+ipfs = "0.2.0"
```
Comment (koivunej) [inline index.html]:
```suggestion
ipfs = "0.2.1"
```

Not sure if "0.2" would suffice. I know "0.2.0" will not work :)

**Review by koivunej — COMMENTED** [review]
I see you already added my suggestions, but I did have this tab open from when I discovered the bug so that might be why you got some errors.

**Review by koivunej — APPROVED** [review]

**Comment by aphelionz** [comment]
Thank you!

**Pull request merged**